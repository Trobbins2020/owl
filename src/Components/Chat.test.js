import Chats from "./Chat";
import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

describe("Chat Component", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Chats />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer.create(<Chats />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
