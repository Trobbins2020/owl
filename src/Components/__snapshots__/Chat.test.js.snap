// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat Component renders the UI as expected 1`] = `
<div>
  <h1>
    Chat Events
  </h1>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Koren Templeton
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <h2>
      6:19:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Koren Templeton
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <h2>
      6:19:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Koren Templeton
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <h2>
      6:19:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Caty Flucker
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <h2>
      6:18:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Axe Kubicka
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
    <h2>
      6:18:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Axe Kubicka
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
    <h2>
      6:21:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Axe Kubicka
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
    <h2>
      6:22:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Frank Runciman
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <h2>
      6:18:04 pm
    </h2>
  </div>
  <div
    className="Participant"
  >
    <p>
      Name: 
      Ashla Attwool
    </p>
    <p>
       inSession: 
      False
    </p>
    <img
      alt="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <h2>
      6:20:04 pm
    </h2>
  </div>
</div>
`;
